eskip/string.go

```diff
@@ -103,15 +105,23 @@ func (r *Route) backendString() string {
 
 // Serializes a route expression. Omits the route id if any.
 func (r *Route) String() string {
+	return r.Print(false)
+}
+
+func (r *Route) Print(pretty bool) string {
```

Comment:
i like it. Can we please do the same for `func String(routes ...*Route) string` to stay a bit consistent?